integration/helpers/loadbalancer/main.go

```diff
@@ -0,0 +1,110 @@
+package main
```

Comment:
Maybe we can remove: https://github.com/chef/automate/blob/master/tools/proxy/main.go